### PR TITLE
Add decrement item call

### PR DESCRIPTION
**Pull request by Protonull**
+ (plus a small improvement to item comparisons)

### CODE REVIEW

**Comment by Protonull** [comment]
Made for [Citadel issue 28](https://github.com/CivClassic/Citadel/issues/28)

**Comment by Maxopoly** [comment]
As discussed in Discord this does not help the Citadel use case. Do you think it's worth merging anyway?

**Comment by Protonull** [comment]
Yeah, I think so. It'll be useful for OldEnchanting in any case.

File: src/main/java/vg/civcraft/mc/civmodcore/api/ItemAPI.java [hunk 1]
```diff
@@ -57,11 +57,17 @@ public static boolean isValidItem(@Nullable ItemStack item) {
 	 * @see ItemStack#equals(Object)
 	 * */
 	public static boolean areItemsEqual(@Nullable ItemStack former, @Nullable ItemStack latter) {
```
Comment (Maxopoly) [inline src/main/java/vg/civcraft/mc/civmodcore/api/ItemAPI.java:59]: Is it intended that this returns false if both items are null?

Comment (Protonull) [reply src/main/java/vg/civcraft/mc/civmodcore/api/ItemAPI.java:59]: Yeah, it says so in the javadoc

**Pull request merged**